test/BUILD.gn

```diff
@@ -146,6 +146,11 @@ test("brave_unit_tests") {
     "//components/sync/service/sync_auth_manager_unittest.cc",
     "//components/sync_device_info/device_info_sync_bridge_unittest.cc",
   ]
+
+  if (!is_android) {
```

Comment:
pls add a source set to browser/ipfs/BUILD.gn instead

Reply:
I just removed that line